### ISSUE TITLE
OCS-Install button not visible on 1280x1024 monitor

### ISSUE THREAD

**Issue by probonopd** [opendesktop/ocsstore#15]
On a 1280x1024 monitor, the OCS-Install button is not visible.

![screenshot from 2017-10-29 19-18-50](https://user-images.githubusercontent.com/2480569/32146895-6e9f9a82-bcde-11e7-9401-99c1ab011f39.png)

**Comment by probonopd** [comment]
@star-buck @rvs75 am I the only one who is annoyed by this?
The install buttons are not shown if the window width is smaller than, well, basically Full HD. At the same time, there is plenty of whitespace at the left and right of the download file area.

In the desktop app, there is __no way__ to scroll horizontally:

![](https://user-images.githubusercontent.com/2480569/35168273-3c1d77f2-fd50-11e7-9098-1014caaee87d.png)

**Comment by star-buck** [comment]
Yes, we need to make the whole design flexible. There is also two buttons
now at the right sidebar for each product, those should be visible?

On Jan 19, 2018 8:39 PM, "probonopd" <notifications@github.com> wrote:

@star-buck <https://github.com/star-buck> @rvs75 <https://github.com/rvs75>
am I the only one who is annoyed by this?
The install buttons are not shown if the window width is smaller than,
well, basically Full HD. At the same time, there is plenty of whitespace at
the left and right of the download file area.

In the desktop app, there is *no way* to scroll horizontally:

<https://user-images.githubusercontent.com/2480569/35168273-3c1d77f2-fd50-11e7-9098-1014caaee87d.png>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/opendesktop/opendesktop-app/issues/15#issuecomment-359068350>,
or mute the thread
